sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JacksonUtils.scala

```diff
@@ -55,4 +58,15 @@ object JacksonUtils {
 
     schema.foreach(field => verifyType(field.name, field.dataType))
   }
+
+  /** Convert a map literal to Map-type data. */
+  def validateMapData(exp: Expression): Map[String, String] = exp match {
```

Comment:
Why we need to keep here? This is not related to `JacksonUtils.scala`. The function name also needs a change.